wondrous-app/components/Common/TaskViewModal/TaskViewModalComponent.tsx

```diff
@@ -437,7 +437,7 @@ export const TaskViewModal = ({
     permissions.includes(PERMISSIONS.EDIT_TASK) ||
     (fetchedTask?.createdBy === user?.id && fetchedTask?.type === TASK_TYPE);
 
-  const showAssignee = !isTaskProposal && !isMilestone && !isBounty;
+  // const showAssignee = !isTaskProposal && !isMilestone && !isBounty;
```

Comment:
We shouldn't comment this out - assignees don't make sense for these other entities